types/unist/index.d.ts

```diff
@@ -6,13 +6,14 @@
 //                 Titus Wormer <https://github.com/wooorm>
 //                 Junyoung Choi <https://github.com/rokt33r>
 //                 Ben Moon <https://github.com/GuiltyDolphin>
+//                 JounQin <https://github.com/JounQin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
 /**
  * Syntactic units in unist syntax trees are called nodes.
  */
-export interface Node {
+export interface Node<T extends object = Data> {
```

Comment:
similar to your other refactor, `NodeData` or `NData` could be clearer here

```suggestion
export interface Node<NData extends object = Data> {
```